impedance/tests/test_circuit_elements.py

```diff
@@ -46,20 +44,18 @@ def test_all():
             assert np.isclose(val, correct_vals[key]).all()
```

Comment:
When I run this test in a Jupyter Lab instance I get an error. The first related to `'__builtins__'` in `circuit_elements`:
```
AttributeError: 'dict' object has no attribute 'num_params'
```
I don't get this same error running pytest locally so maybe Jupyter isn't handling the globals appropriately?